portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/FormEvents.testcase

```diff
@@ -65,6 +65,171 @@ definition {
 		ACUtils.tearDownAC();
 	}
 
+	@description = "Bug: LRAC-7832 | Automation ID: LRAC-9867 | Test Summary: Check fieldBlurred and fieldFocused is triggered when interacting with form fields (Text, Date, Date and Time, Image, Rich Text, Upload and Color[by color picker and color field])"
+	@priority = "4"
+	test CheckFieldBlurredAndFieldFocusedWhenInteractingWithFields {
+		property proxy.server.enabled = "true";
+
+		var fieldNameList = "Date,Image,Rich Text,Upload,Color,Date and Time";
+
+		task ("Edit the form to add more fields") {
+			FormsAdmin.openFormsAdmin(siteURLKey = "site-name");
+
+			FormsAdminNavigator.gotoEditFormByName(formName = "Created Form Name");
+
+			DataEngine.addFields(fieldNameList = "${fieldNameList}");
+
+			Form.save();
+		}
+
+		task ("Navigate to view the form page") {
+			ACUtils.navigateToSitePage(
+				actionType = "View Form",
+				pageName = "AC Page",
+				siteName = "Site Name");
+		}
+
+		task ("This loop will interact with the input field of all fields") {
+			for (var fieldName : list "Text,${fieldNameList}") {
+				task ("Start Har recording") {
+					ProxyUtil.startHarRecording("fieldFocused");
+				}
+
+				task ("Click to focus on form fields") {
+					if ("${fieldName}" == "Date and Time") {
+						Click(
+							key_fieldName = "DateAndTime",
+							locator1 = "FormFields#GENERIC_INPUT_FIELD");
+					}
+					else if ("${fieldName}" == "Rich Text") {
+						FormFields.addRichTextContent(
+							content = "",
+							fieldLabel = "Rich Text");
+					}
+					else if (("${fieldName}" != "Rich Text") && ("${fieldName}" != "Date and Time")) {
+						Click(
+							key_fieldName = "${fieldName}",
+							locator1 = "FormFields#GENERIC_INPUT_FIELD");
+					}
+				}
+
+				task ("Wait for sending requests") {
```

Comment:
This pause and the other ones are fine for now but later we should refactor this into a macro

Reply:
Yes, I left the pauses for us to investigate and resolve them in the ticket you opened, I think this makes it easier for us to organize and resolved. Thanks, Chris.